db/migrate/20170327105842_change_max_delivery_time.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeMaxDeliveryTime < ActiveRecord::Migration[5.0]
+  def change
+  	reversible do |dir|
```

Comment:
Cool! I think it's more common here to implement `up` and `down`, it's no big deal that you repeat `change_table :restaurants`

Did you try to migrate and rollback, and see if this works?